cea/inputlocator.py

```diff
@@ -468,6 +474,33 @@ def get_edge_mass_flow_csv_file(self, network_type):
         return os.path.join(self.get_optimization_network_layout_folder(), 'NominalEdgeMassFlow_' +
                              network_type + '.csv')
 
+    def get_network_layout_edges_shapefile(self, network):
```

Comment:
I suggest to change the input folder name to "thermal-networks". It makes it easier to differentiate with prospect electric networks.

Inside the folder of thermal-networks I suggest to include the shapefiles without any extra subfolders. it would make easier to write documentation. an example for three types of networks.

- type1_nodes.shp
- type1_edges.shp
- type2_nodes.shp
- type2_edges.shp
- type3_nodes.shp
- type3_edges.shp